### PR TITLE
Update dependency resolutions to fix Dependabot issues

### PR DESCRIPTION
**Pull request by ivarpruijn**
See https://github.com/objectiv/objectiv.io/security/dependabot

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -34,8 +34,9 @@
     "trim": "^0.0.3",
     "nth-check": "2.0.1",
     "node-fetch": "^3.2.0",
-    "node-forge": "^1.2.1",
+    "node-forge": "^1.3.0",
     "prismjs": "^1.27.0",
+    "minimist": "^1.2.6",
     "@braintree/sanitize-url": "^6.0.0"
   },
```
Comment (sdirosa) [inline docs/package.json:41]: 👌🏼

**Review by sdirosa — APPROVED** [review]
LGTM

File: docs/package.json (hunk 1 above)
Comment (sdirosa) [reply docs/package.json:41]: I guess at some point we should double check this resolution list, because probably some of them have been solved

Comment (ivarpruijn) [reply docs/package.json:41]: Good point. I'll pick that up on the next Docusaurus upgrade that I'll do soon, will make a note.

**Pull request merged**